frontend/src/views/AppView.vue

```diff
@@ -141,6 +153,148 @@ const copyURL = () => {
     });
 };
 
+const config = ref(searchConfigStore.searchConfig);
+watch(
+  () => config, // ref to searchConfigStore.searchConfig
```

Comment:
`watch(config, ...)`
we can watch config directly if it is already a Ref or computed obj.